lock.go

```diff
@@ -13,35 +14,52 @@ import (
 const luaRefresh = `if redis.call("get", KEYS[1]) == ARGV[1] then return redis.call("pexpire", KEYS[1], ARGV[2]) else return 0 end`
 const luaRelease = `if redis.call("get", KEYS[1]) == ARGV[1] then return redis.call("del", KEYS[1]) else return 0 end`
 
+var ErrCanntGetLock = errors.New("can't get lock")
```

Comment:
could you pls rename this to `ErrCannotGetLock = errors.New("cannot get lock")`

Reply:
hm.. no problems